test/parallel/test-force-repl.js

```diff
@@ -4,12 +4,20 @@ const assert = require('assert');
 const spawn = require('child_process').spawn;
 
 // Spawn a node child process in interactive mode (enabling the REPL) and
-// confirm the '> ' prompt is included in the output.
+// confirm the '> ' prompt and welcome message is included in the output.
 const cp = spawn(process.execPath, ['-i']);
 
 cp.stdout.setEncoding('utf8');
 
-cp.stdout.once('data', common.mustCall(function(b) {
-  assert.strictEqual(b, '> ');
-  cp.kill();
+let out = '';
+cp.stdout.on('data', (d) => {
+  out += d;
+});
+
+cp.stdout.on('end', common.mustCall(() => {
+  assert.strictEqual(out, `Welcome to Node.js ${process.version}.\n` +
+                        'Type ".help" for more information.\n> ');
+  console.log('ok');
```

Comment:
Is there a specific reason that the `console.log('ok');` is added here?

Reply:
Looks like I used it fore debug back then, I will remove it.